elements/rh-back-to-top/docs/10-overview.md

```diff
@@ -0,0 +1,20 @@
+## Overview
+{{ tagName | getElementDescription }}
+
+
+
+## Demos
+  View a live version of this component and see how it can be customized.
+  {% playground tagName=tagName %}{% endplayground %}
+  {% cta href="./demo/", target="_blank" %}
+    View the `<rh-back-to-top>` demo in a new tab
+  {% endcta %}
+
+
+## When to use
+  - 
+
+{% repoStatus type="Element" %}
```

Comment:
Coordinate with diwanshi's pr

Reply:
will rebase this when #1502 merges

Reply:
There was a small bug in the repoStatus shortcode where if a component didn't yet exist in the repoStatus.yml it would fail.  I fixed this here, but in order to get this update into the other staging/chansey PR's this will need to merge first.